deploy_prerequisites.yml

```diff
@@ -16,6 +16,6 @@
 
 - import_playbook: playbooks/deploy_assisted_installer_onprem.yml
 
-- import_playbook: playbooks/create_vms.yml
+- import_playbook: playbooks/kvm/create.yml
```

Comment:
Could you also factor out host network configuration as we need that to happen at the start of this play book as it will be required for some deployment scenarios.